README.md

```diff
@@ -282,26 +282,45 @@ Example:
           success: function ({getState, dispatch, getSourceAction}, req) {
             console.log(req); //contains information about request object
             //...
+            return req;
           },
           error: function ({getState, dispatch, getSourceAction}, error) {
             //...
+            return response
```

Comment:
Typo - should this be `return Promise.reject(error);` ?

Reply:
I think so but let me test to make sure.